pyproject.toml

```diff
@@ -17,7 +17,7 @@ packages = [
 
 [tool.poetry.dependencies]
 python = "^3.10"
-pyteal = "~0.22.0"
+pyteal = {git = "https://github.com/algorand/pyteal", branch = "asdict4BareCallActions"}
```

Comment:
Don't merge until this can be set to an actual version number

Reply:
cant even CI it til then :(

Reply:
are we good now that I've pinned to "~0.23.0" ?